src/sentry/plugins/bases/notify.py

```diff
@@ -129,7 +129,13 @@ def __is_rate_limited(self, group, event):
             limit=10,
         )
 
+    def is_configured(self, project):
```

Comment:
:+1: Then luckily, yeah, every notification plugin we run has this implemented as expected.